### PR TITLE
New dart client API

### PR DESCRIPTION
**Pull request by dmdashenkov**
This PR migrates the Dart client to the new Dart library of version 1.7.0.

The Web interface also now exposes the subscription endpoints so that the Dart client may subscribe to events.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

File: web/src/main/java/io/spine/examples/blog/web/BlogCancelServlet.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import io.spine.core.Response;
+import io.spine.web.subscription.servlet.SubscriptionCancelServlet;
+
+import javax.servlet.annotation.WebServlet;
+
+import static io.spine.examples.blog.web.Application.app;
+
+@SuppressWarnings("serial")
+@WebServlet("/subscription/cancel")
+public final class BlogCancelServlet extends SubscriptionCancelServlet<Response> {
```
Comment (armiol) [inline web/src/main/java/io/spine/examples/blog/web/BlogCancelServlet.java:38]: Please document this type.

File: web/src/main/java/io/spine/examples/blog/web/BlogKeepUpServlet.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import io.spine.core.Response;
+import io.spine.web.subscription.servlet.SubscriptionKeepUpServlet;
+
+import javax.servlet.annotation.WebServlet;
+
+import static io.spine.examples.blog.web.Application.app;
+
+@SuppressWarnings("serial")
+@WebServlet("/subscription/keep-up")
+public final class BlogKeepUpServlet extends SubscriptionKeepUpServlet<Response> {
```
Comment (armiol) [inline web/src/main/java/io/spine/examples/blog/web/BlogKeepUpServlet.java:38]: Same as above.

**Review by armiol — CHANGES_REQUESTED** [review]
@dmdashenkov LGTM in general. Please fill in the missing pieces of the documentation. Some of newly created types aren't supplied with Javadocs.

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

**Review by armiol — APPROVED** [review]

**Pull request merged**